images/dind/Dockerfile

```diff
@@ -72,6 +78,8 @@ RUN systemctl enable docker.service
 RUN echo "DOCKER_STORAGE_OPTIONS=--storage-driver vfs" >\
  /etc/sysconfig/docker-storage
 
+COPY openshift-dind-lib.sh /usr/local/bin/
```

Comment:
Why is this moving from the node image?  My intent was to allow the base dind image to be used for things other than running an image-based openshift cluster (e.g. as an ansible target).